### PR TITLE
22225: Add links to accessibility guides section

### LINKED ISSUE #22225: ARIA Guides page has no links to any guides, just titles
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Guides

### What specific section or headline is this issue about?

ARIA guides

### What information was incorrect, unhelpful, or incomplete?

There is a list of guides that seems like it should link out the various guides, but they're just text

### What did you expect to see?

Links to the guides so I can learn more about proper ARIA usage

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_guides`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Guides
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/aria_guides/index.md
* Last commit: https://github.com/mdn/content/commit/95cdbf44bd47782a5afb6ec4a3c4d00a5a7958e3
* Document last modified: 2022-10-03T02:59:57.000Z

</details>

### PR DESCRIPTION
**Pull request by bsmth**
The ARIA guides section was a text-only list. After going back through the project history, I cannot find the associated links with the text. This PR adds a list of resources to get started on refreshing this.

Fixes #22225

### CODE REVIEW

**Comment by bsmth** [comment]
Hi @dipikabh would you mind having a look when you get the chance?

File: files/en-us/web/accessibility/aria/aria_guides/index.md [hunk 1]
```diff
@@ -27,20 +27,23 @@ tags:
   </ol>
 </section>
 
-Accessible Rich Internet Applications (**ARIA**) defines ways to make the Web more accessible to people with disabilities. A few guidelines to follow that ensure better accessibility, like using drag and drop for placement of the widgets.
+Accessible Rich Internet Applications (**ARIA**) defines ways to make the Web more accessible to people with disabilities.
```
Comment (dipikabh) [inline files/en-us/web/accessibility/aria/aria_guides/index.md:30]:
```suggestion
Accessible Rich Internet Applications (**ARIA**) define ways to make the web more accessible to people with disabilities.
```

File: files/en-us/web/accessibility/aria/aria_guides/index.md [hunk 2]
```diff
@@ -27,20 +27,23 @@ tags:
...
+This page lists guides that can help improve accessibility, such as designing tables and forms, keyboard-navigation, testing, and more.
```
Comment (dipikabh) [inline files/en-us/web/accessibility/aria/aria_guides/index.md:31]:
```suggestion
This page lists the guides that you can use to help improve the accessibility of web page features such as tables, forms, and keyboard-navigation.
```

File: files/en-us/web/accessibility/aria/aria_guides/index.md [hunk 3]
```diff
@@ -27,20 +27,23 @@ tags:
...
-- Using ARIA with HTML5
-- How to test ARIA
-- ARIA on Mobile Devices
+- [Designing accessible forms](https://www.w3.org/WAI/tutorials/forms/)
+- [WAI-ARIA basics](/en-US/docs/Learn/Accessibility/WAI-ARIA_basics)
+- [Keyboard-navigable JavaScript widgets](/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets)
+- [Using ARIA for labels with embedded fields inside them](/en-US/docs/Web/Accessibility/ARIA/forms/Multipart_labels)
+- [Managing Focus in Composite Widgets](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/) (`aria-activedescendant` vs roving tabindex)
+- [How to test ARIA](https://www.w3.org/WAI/test-evaluate/)
+- [Displaying accessible tables](https://www.w3.org/WAI/tutorials/tables/)
```
Comment (dipikabh) [inline files/en-us/web/accessibility/aria/aria_guides/index.md:39]:
```suggestion
- [Designing accessible forms](https://www.w3.org/WAI/tutorials/forms/)
- [Understanding WAI-ARIA basics](/en-US/docs/Learn/Accessibility/WAI-ARIA_basics)
- [Creating keyboard-navigable JavaScript widgets](/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets)
- [Using ARIA for labels with embedded fields inside them](/en-US/docs/Web/Accessibility/ARIA/forms/Multipart_labels)
- [Managing focus in composite widgets](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/) (`aria-activedescendant` vs roving tabindex)
- [Testing ARIA](https://www.w3.org/WAI/test-evaluate/)
- [Displaying accessible tables](https://www.w3.org/WAI/tutorials/tables/)
```
With the suggested changes above, the list will be parallel in sentence construction and case but will not match the titles on the linked pages. You can take a call whether to accept or reject these suggestions.

File: files/en-us/web/accessibility/aria/aria_guides/index.md [hunk 4]
```diff
@@ -27,20 +27,23 @@ tags:
...
+
+## See also
+
+The following references are helpful for common accessibility use case
```
Comment (dipikabh) [inline files/en-us/web/accessibility/aria/aria_guides/index.md:43]:
```suggestion
The following references are helpful for common accessibility use cases:
```

File: files/en-us/web/accessibility/aria/aria_guides/index.md [hunk 5]
```diff
@@ -27,20 +27,23 @@ tags:
...
+
+- [Labeling widgets](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)
+- [Landmark Roles](/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role)
```
Comment (dipikabh) [inline files/en-us/web/accessibility/aria/aria_guides/index.md:46]:
```suggestion
- [Landmark roles](/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role)
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Just a few nits. Apologies for the delay.

File: files/en-us/web/accessibility/aria/aria_guides/index.md (hunk 3 above)
Comment (bsmth) [reply files/en-us/web/accessibility/aria/aria_guides/index.md:39]: This is good, tnx 👍🏻

**Comment by bsmth** [comment]
Thank you, suggestions look good 👍🏻

**Review by dipikabh — APPROVED** [review]
Thank you for the updates!

**Pull request merged**